package-lock.json

```diff
@@ -1,8 +1,8361 @@
 {
 	"name": "vscode-camelk",
 	"version": "0.0.32",
-	"lockfileVersion": 1,
+	"lockfileVersion": 2,
```

Comment:
woudl be better to upgrade the lock file version in a separated Pull Request

Reply:
You are right, I've removed unnecessary package-lock.json changes from this PR